vertical-pod-autoscaler/deploy/kustomization.yaml

```diff
@@ -3,4 +3,4 @@ resources:
 - recommender-deployment.yaml
 - updater-deployment.yaml
 - vpa-rbac.yaml
-- vpa-v1-crd.yaml
+- vpa-v1-crd-v1-gen.yaml
```

Comment:
vpa-v1-crd-gen

Reply:
Recheck all file names and unified to vpa-v1-crd-gen. Also, recreated the CRD manifest.